mllib/src/main/scala/org/apache/spark/mllib/clustering/LDAModel.scala

```diff
@@ -345,6 +346,12 @@ class DistributedLDAModel private (
     }
   }
 
+  /** Java-friendly version of [[topicDistributions]] */
+  def javaTopicDistributions: JavaPairRDD[java.lang.Long, Vector] = {
+    new JavaPairRDD[java.lang.Long, Vector](
```

Comment:
How is that different?  Looking at the definition of JavaPairRDD.fromRDD, it looks like exactly the same call.